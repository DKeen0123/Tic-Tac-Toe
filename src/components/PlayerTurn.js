import React from 'react';

const PlayerTurn = () => {
  return (
    <div>
      <h3>Player 1's turn!</h3>
    </div>
  );
};

export default PlayerTurn;
