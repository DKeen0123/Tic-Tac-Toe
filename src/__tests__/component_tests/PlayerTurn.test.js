import React from 'react';
import { shallow } from 'enzyme';
import PlayerTurn from '../../components/PlayerTurn';

describe('PlayerTurn', () => {
  const playerTurn = shallow(<PlayerTurn />);

  it('renders correctly', () => {
    expect(playerTurn).toMatchSnapshot();
  });
});
