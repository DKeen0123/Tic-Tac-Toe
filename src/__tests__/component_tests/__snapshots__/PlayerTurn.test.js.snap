// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerTurn renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PlayerTurn />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h3>
        Player 1's turn!
</h3>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Player 1's turn!",
      },
      "ref": null,
      "rendered": "Player 1's turn!",
      "type": "h3",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h3>
          Player 1's turn!
</h3>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Player 1's turn!",
        },
        "ref": null,
        "rendered": "Player 1's turn!",
        "type": "h3",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
